cime_config/config_grids.xml

```diff
@@ -4417,6 +4435,13 @@
       <map name="ATM2ROF_SMAPNAME">cpl/gridmaps/antarcticax4v1np4/map_antarcticax4v1pg2_to_r0125_mono.20200925.nc</map>
     </gridmap>
 
+    <gridmap atm_grid="ne0np4_CAx32v1.pg2" rof_grid="r0125">
+     <map name="ATM2ROF_FMAPNAME">cpl/gridmaps/CAx32v1pg2/map_CAne32x32v1pg2_to_r0125_mono.20220430.nc</map>
```

Comment:
These maps should be renamed to follow our convention: https://acme-climate.atlassian.net/wiki/spaces/DOC/pages/1217757434/Mapping+file+algorithms+and+naming+convention

Reply:
Thanks, will do!